tiledb/sm/query/writer_base.cc

```diff
@@ -945,7 +947,8 @@ Status WriterBase::new_fragment_name(
   frag_uri->clear();
   RETURN_NOT_OK(uuid::generate_uuid(&uuid, false));
   std::stringstream ss;
-  ss << "/__" << timestamp << "_" << timestamp << "_" << uuid << "_"
+  ss << "/__" << std::setfill('0') << std::setw(13) << timestamp << "_"
```

Comment:
What is happening here?

Reply:
same as in metadata.cc , filename time-sequence sorting order issue correction.